.buildkite/scripts/steps/serverless/build_and_deploy.sh

```diff
@@ -137,8 +137,35 @@ EOF
   buildkite-agent meta-data set pr_comment:early_comment_job_id "$BUILDKITE_JOB_ID"
 }
 
+# This is the integration with the observability-test-environments
+create_github_issue_oblt_test_environments() {
+
+cat <<EOF > .issue-body
+### Kibana image
+
+$KIBANA_IMAGE
+
+### Kibana pull request
+
+$BUILDKITE_PULL_REQUEST
+
+### Further details
+
+Caused by $GITHUB_PR_TRIGGER_USER using the github label in $BUILDKITE_REPO/pull/$BUILDKITE_PULL_REQUEST
+EOF
+
+  gh issue create \
+    --title "[Deploy Serverless Kibana] for user $GITHUB_PR_TRIGGER_USER with PR kibana@pr-$BUILDKITE_PULL_REQUEST" \
+    --body-file ".issue-body" \
+    --label 'deploy-custom-kibana-serverless' \
+    --repo 'elastic/observability-test-environments'
```

Comment:
Shall we assign the PR owner to be notified when the deployment is available?

```suggestion
    --repo 'elastic/observability-test-environments' \
    --assignee "$GITHUB_PR_OWNER"
```

Reply:
I think `$GITHUB_PR_OWNER` is correct for your intention here. For reference, there is an environment tab in Buildkite with most of BK / Github environment variables available on any step:

![image](https://github.com/elastic/kibana/assets/14021797/0389a3c6-fa5d-4fcd-8180-562b80bb4f50)